contracts/pcv/liquity/BAMMDeposit.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;  
```

Comment:
I noticed sometimes we use ^0.8.0 and sometimes ^0.8.4, do you think it matters ? local config is 0.8.10 anyway

Reply:
shouldn't matter much but good to be consistent. We can pick one and update everything

Reply:
we should just lock all contracts to a specific version by doing `pragma solidity 0.8.10` or `pragma solidity =0.8.10` https://github.com/ConsenSys/smart-contract-best-practices/issues/125